[ADD] elearning: Create your course (#669)
odoo/documentation-user: elearning/going_online/create_your_course.rst

or **Members Only**: the user has to be an attendee of the course to see the content and enroll in the course.
You can also choose which content is first shown on your course's page.
**Latests Published**: new content is displayed first.
**Most Voted**: the content your audience liked the most appears as featured.
**Most Viewed**: the content with the most views appears as featured.

.. image:: media/display.png
   :align: center
   :alt: picking the display for your course

The featured content is displayed on top of the course's overview.

.. image:: media/disp.png
   :align: center
   :alt: picking the display for your course

.. note::
   This will only affect the display if you have chosen Documentation as the course type.

Description
-----------

A course description is a brief summary about the content of that course is about. It appears on the website along
with the title, and can be useful to have a quick overview of the course.

.. image:: media/website-desc.png
   :align: center
   :alt: display of the course description on the website

Publish your course
===================

> When you create a course, it is by default unpublished on the Website.
> If the earth icon on your course form is red, that means the course is not yet published.
> Click it to access the website.

Reviewer: ```suggestion
When you create a course, it is by default unpublished on the Website.
```

.. image:: media/publish.png
   :align: center
   :alt: How to publish your course

To publish it, click on *Go to Website* and disable *Unpublished*. Courses can be published or unpublished at any time.

.. image:: media/unpublished.png
   :align: center
   :alt: Unpublished course on the website

Edit a course from the website
==============================

Courses content can be always added through the website, by clicking on *Add Content*.

.. image:: media/edit-website.png
   :align: center
   :alt: How to edit content from the website

> You can also change the layout by clicking on **Edit**. That will open the website builder editor to make changes to
> your course by using the website building blocks.

Reviewer: ```suggestion
You can also change the layout by clicking on **Edit** to open the website builder editor. Make changes to
your course by dragging & dropping building blocks.
```

==================
Create your course
==================

The Odoo eLearning application allows you to create courses and get them online and running in minutes thanks to the
friendly Odoo interface.

Configuration
=============

Go to :menuselection:`eLearning → Dashboard`. The course overview is the default dashboard for the eLearning

> Application. You can also access course creation via :menuselection:`eLearning → Courses→ Courses`.

Author: changed it to access course creation

Create course
=============

From the dashboard, you have access to the already created courses. To create a new course, click the **create** button
on top.

.. image:: media/create-course.png
   :align: center
   :alt: Elearning Overveview

The course form will open. From here you can configure your course, give him a title and start adding content.
Complete the following fields for your new course.

*  **Course title** pick a name for your course.
*  **Tags** Create and edit tags for your course, this can help you and students find courses with keywords. You can use
   already created tags or create them as you go.

> * **Image** upload a png file by clicking the photo logo to illustrate your course.

Author: changed it to image file

.. image:: media/course-config.png
   :align: center
   :alt: course-config

Course Configuration
====================

You can configure your course when you create it or always go back after and edit the configuration for each course with
the edit button.
Don’t forget to always **save** when making changes to a course.

.. image:: media/edit-course.png
   :align: center
   :alt: editing your course

Options
-------

Here you will find the most important configuration for your course.

Course type
~~~~~~~~~~~

First you can choose the course type. This will change the course display on the website.

.. image:: media/type-documentation.png
   :align: center
   :alt: course-config

**Training** courses will also allow you to track the progress of your students while **Documentation** is mainly used

> when contents within the course don’t specially follow each other. Here is the same course with the different displays.

Author: I wanted to show how different displays affected the same course to have a better comparison

> when contents within the course don’t specially follow each other. Here is the same course with the different displays.

Author: But the phrasing is weird I'm changing it

**Documentation**

.. image:: media/type-training.png
   :align: center
   :alt: course-config

**Training**

.. image:: media/type-training2.png
   :align: center
   :alt: course-config

Course responsible
~~~~~~~~~~~~~~~~~~

For each course you have to choose a responsible. The course responsible will have all accesses regarding the course.
He will be able to see the list of attendees, send them emails, add content...

.. image:: media/course-resp.png
   :align: center
   :alt: course-config

> The course responsible can be changed by clicking the arrow next to their name at any moment.
> If you have multiple websites you can choose which website your course is going to be displayed on.

Reviewer: ```suggestion
If you have multiple websites you can choose at which website your course is going to be displayed on.
```

Access Rights
~~~~~~~~~~~~~

Access rights will allow you to decide who can access your courses. If you choose **Public**, anyone who finds the
course on your website will be able to enroll and access your content.

.. image:: media/enroll-policy.png
   :align: center
   :alt: picking the enroll policy

> You can send invite mails directly from the dashboard of the eLearning App.
> Go to the settings button of the course you want to invite people to and click on invite.
> This will allow you to send a mail inviting people to your course.

Reviewer: ```suggestion
You can send invite emails directly from the course's form, or from the dashboard, by going to *Settings* and selecting *Invite*.
```

.. image:: media/invite.png
   :align: center
   :alt: invite students to your course

> When choosing the **Enroll Policy** **On Invitation** a smart **invite** button will appear on the course. This will
> allow you to send mails to students so they can enroll.

Reviewer: ```suggestion
```

> 

Reviewer: You can also invite people to attend the course by click on the *Share* button on the course's website page.  A link to
access the course is generated, and can be shared as many time was you like.

You also have the possibility to directly share the course on **Social Networks** with our smart buttons. -> I think this paragraph is weird here, because the user does not know how to publish the course yet, meaning that he does not know how to see this.  The first of the paragraph make make sense being here, but this second, could be displayed after the of shows how to publish a course?

.. image:: media/share.png
   :align: center
   :alt: sharing your course

You can also invite people from the course on the website with the share button who will provide you with a link to
access the course, you can then copy the text and send it to whoever you want to invite to the course. You also have
the possibility to directly share the course on **Social Networks** with our smart buttons.

.. image:: media/social.png
   :align: center
   :alt: sharing the course on social networks

Display
~~~~~~~

> If the course is public, everyone will be able to see and access the course. When the display is set to members only,
> you have to be an attendee of the course to see the contents and enroll in the course.
> You can also choose which content will be shown first in your course.
> **Latests Published** new content will always appear first.
> **Most Voted** the content that you audience liked the most will appear as featured.
> **Most Viewed** the content that has the most views will appear as featured.

Reviewer: ```suggestion
A *Public* course is visible and can be accessed by anyone. When the display is set to *Members Only*,
you have to be an attendee of the course to see the content and enroll in the course.
You can also choose which content is first shown in your course.
**Latests Published**: new content is displayed first.
**Most Voted**: the content your audience liked the most appears as featured.
**Most Viewed**: the content that has the most views appears as featured.
```

.. image:: media/display.png
   :align: center
   :alt: picking the display for your course

> Here is an example where you can see the featured content above the rest of the course content.

Reviewer: An example of what? Latest published, most voted, or most viewed?

.. image:: media/disp.png
   :align: center
   :alt: picking the display for your course

.. note::

>    This will only affect the display if you have chosen Documentation as course type.

Reviewer: ```suggestion
   This will only affect the display if you have chosen Documentation as the course type.
```

> Description

Reviewer: I'd keep the order of the titles as the order of the tabs. Meaning, first the subtitle Description, and then Options in the doc (:

-----------

> A course description is a brief summary of the content that’s in your course. Course descriptions appear on the Website
> along the title of the course. They can be very useful for students to understand quickly what the course is about.

Reviewer: ```suggestion
A course description is a brief summary of what the content of that course is about. They appear on the website along with the title and can be useful to have a quick overview of the course.
```
or something like this..

.. image:: media/description.png
   :align: center
   :alt: course description for your course

The course description will appear under the title of your course on the website and once the course is published

> everyone enrolling will have access to the description.

Reviewer: this is said in the paragraph above!?
Idea: delete the first image and second paragraph, and keep the first paragraph with the second image.

.. image:: media/website-desc.png
   :align: center
   :alt: display of the course description on the website

Publish your course
===================

> When you create a course it is per default unpublished and will not be seen on the Website.
> If the earth symbol on your course is in red, that means the course is not published.
> Click it to access the website.

Reviewer: ```suggestion
When you create a course, it is by default unpublished on the Website.
If the earth icon on your course form is red, that means the course is not published.
```

.. image:: media/publish.png
   :align: center
   :alt: How to publish your course

> When you are on the website you will find a button to publish your course on top of the page.
> Thanks to this button you can publish or unpublish a course in an instant.

Reviewer: ```suggestion
To publish it, click on *Go to Website* and disable *Unpublished*. Courses can be published or unpublished at any time.
```

.. image:: media/unpublished.png
   :align: center
   :alt: Unpublished course on the website

> Edit from the Website

Reviewer: ```suggestion
Edit a course from the website
```

=====================

> Once a course is created you can always edit it directly from the website, adding content

Reviewer: ```suggestion
Courses' content can be always added through the website, by clicking on *Add Content*.
```

.. image:: media/edit-website.png
   :align: center
   :alt: How to edit content from the website

> You can also change the layout with ease thanks to our building blocks.
> Click the **edit** button on top of the website to start making changes to your course.

Reviewer: ```suggestion
You can also change the layout by clicking on **Edit**. That will open the website builder editor and ...
```

or something like this (and improving).

==================

The Odoo eLearning application allows you to create courses and get them online and running in minutes thanks to the
friendly Odoo interface. Online learning has been on the rise in recent years, and it’s not really hard to see why.
eLearning courses are more convenient than traditional face to face courses. Students can fit them around their existing
responsibilities and commitments, allowing them to engage with content and extra learning materials whenever is most
convenient to them.

Access course creation
======================

Go to :menuselection:`eLearning --> Dashboard`. The course overview is the default dashboard for the eLearning
Application. You can also access course creation via :menuselection:`eLearning --> Courses --> Courses`.

> Create courses

Author: I think this works fine

> Course type

Author: I believe it's ok

~~~~~~~~~~~

> Choose between *Documentation* and *Training* as the course type. They are displayed differently on the website.

Author: I will keep the titles for each tab it makes it easier

Access Rights
~~~~~~~~~~~~~

Access rights allow you to decide which users can have access to your courses. If you choose **Public**, anyone who
finds the course on your website is able to enroll and access the content. **On Invitation** on the other hand, requires

> you to send an email invitation to users to be able to enroll.

Author: invitation email indeed

.. image:: media/enroll-policy.png
   :align: center
   :alt: picking the enroll policy

You can send invite emails directly from the course's form, or from the dashboard, by going to *Settings* and selecting

> *Invite*

Author: Yup!

Display
~~~~~~~

A *Public* course is visible and can be accessed by anyone. When the display is set to *Members Only*, you have to be an
attendee of the course to see the content and enroll in the course.
You can also choose which content is first shown in your course.
**Latests Published** new content is displayed first.
**Most Voted** the content your audience liked the most appears as featured.
**Most Viewed** the content with the most views appears as featured.

.. image:: media/display.png
   :align: center
   :alt: picking the display for your course

> This is how the featured content displays above the rest of the course content.

Reviewer: ```suggestion
The featured content is displayed on top of the course's content.
```

.. image:: media/disp.png
   :align: center
   :alt: picking the display for your course

.. note::
   This will only affect the display if you have chosen Documentation as the course type.

Description
-----------

> A course description is a brief summary of what the content of that course is about. They appear on the website along
> with the title and can be useful to have a quick overview of the course.

Reviewer: ```suggestion
A course description is a brief summary about what the content of that course is about. It appears on the website along
with the title, and can be useful to have a quick overview of the course.
```

Publish your course
===================

When you create a course, it is by default unpublished on the Website.
If the earth icon on your course form is red, that means the course is not published.

> Click it to access the website.

Reviewer: ```suggestion
When you create a course, it is by default unpublished on the Website. As a sign of it, the
earth icon on your course form is red. Click on it to access the website.
```

Author: I prefer the ways it's now

.. image:: media/publish.png
   :align: center
   :alt: How to publish your course

> To publish it, click on *Go to Website* and disable *Unpublished*. Courses can be published or unpublished at any time.

Reviewer: ```suggestion
Now, disable *Unpublished*. Courses can be published or unpublished at any time.
```

Author: I prefer it this way

.. image:: media/unpublished.png
   :align: center
   :alt: Unpublished course on the website

Edit a course from the website
==============================

Courses content can be always added through the website, by clicking on *Add Content*.

.. image:: media/edit-website.png
   :align: center
   :alt: How to edit content from the website

> You can also change the layout by clicking on **Edit**. That will open the website builder editor to make changes to
> your course.

Reviewer: ```suggestion
You can also change the layout of your page by clicking on **Edit**. That opens the website builder editor allowing you to make changes to your course's page by dragging and dropping blocks.
```

create a website builder content?

Author: Added the Website doc link
